assert/assertions.go

```diff
@@ -1318,6 +1319,20 @@ func DirExists(t TestingT, path string, msgAndArgs ...interface{}) bool {
 	return true
 }
 
+// EqualFileContent checks whether a file has the expected content.
+func EqualFileContent(t TestingT, path string, content string, msgAndArgs ...interface{}) bool {
```

Comment:
IMO 'EqualFileContent' implies comparing the contents of two files. Would it not be clearer if the function was named 'FileTextContents' ?